### PR TITLE
Amaltest1

### PR DESCRIPTION
**Pull request by wamarock**
please check this PR

### CODE REVIEW

File: src/tests/smoke.test.ts
Comment (madhu9205) [inline src/tests/smoke.test.ts:1]: plz remove the old test

**Review by madhu9205 — CHANGES_REQUESTED** [review]
plz remove the old test and fix the formatting issues.

**Comment by wamarock** [comment]
Could you please check now

**Review by madhu9205 — APPROVED** [review]
lgtm approved.

**Pull request merged**